justfile

```diff
@@ -47,13 +47,13 @@ release *ARGS:
 test:
     cargo test --no-fail-fast --all-features --all-targets
 
-integration-test:
-    #!/usr/bin/env bash
-    ci/build_scripts/build.sh x86_64-unknown-linux-musl
-    cd tests/RobotFramework
-    source .venv/bin/activate
-    invoke build --local
-    invoke tests
+# Install integration test dependencies
```

Comment:
It'd be better if the supported args (the mandatory ones, if any, or the mostly commonly used ones) are also documented so that the user knows what to invoke these recipes with.

Reply:
I wanted to keep the justfile stuff simple for the common use-cases. Since running individual integration tests can be done inside VSCode very conveniently, I thought it didn't really make too much sense to include every possible option in the justfile (as it just introduces too much noise for the common user).

If you want more control over which options are available in the underlying file you can use the python `invoke` task 
runner instead, see example below.

```sh
cd tests/RobotFramework
source .venv/bin/activate
invoke test --help
```

*Output*
```
Usage: inv[oke] [--core-opts] test [--options] [other tasks here ...]

Docstring:
  Run tests

  Examples

      invoke test
      # run all tests

      invoke test --test-name "Successful shell command with output" --processes 1
      # Run any test cases matching a give string

      invoke test --suite "shell_operation" --processes 1
      # Run suites matching a specific name

      invoke test --include "theme:troubleshooting AND theme:c8y" --processes 10
      # Run tests which includes specific tags

Options:
  -a STRING, --adapter=STRING     Default device adapter to use to run tests. e.g. docker, ssh or local
  -e STRING, --exclude=STRING     Don't run tests matching the given tag
  -i STRING, --include=STRING     Only run tests matching the given tag
  -o STRING, --outputdir=STRING   Output directory where the reports will be saved to
  -p STRING, --processes=STRING   Number of processes to use when running tests
  -r INT, --retries=INT           Max global retries to execute on failed tests. Defaults to 0
  -s STRING, --suite=STRING       Only run suites matching the given text
  -t STRING, --test-name=STRING   Only run tests matching the given text
```

The python invoke task runner function is defined [here](Code: https://github.com/thin-edge/thin-edge.io/blob/48f43b5c36e9e8c04c475008b271380b13d7f9e6/tests/RobotFramework/tasks.py#L407)

Reply:
I'm completely aligned with you on keeping the justfile simple so that a normal user can just use it without having to worry about extra arguments and stuff. That's why I was a uncomfortable seeing the ARGS option as I thought that the users would have to dig up and remember what args to use as well. But, if those args are just for advanced customisations and no args are required for the most common usage, then this completely fine.